scripts/check_total_coverage.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
lets have a script called coverage.sh that is able to return unit, e2e or combined

Reply:
can do - if not urgent i'd leave this to my next PR

Reply:
can you rename this to coverage.sh ?

Reply:
or check_coverage.sh

Reply:
done - `coverage.sh`